nio_cli/commands/blockpublish.py

```diff
@@ -0,0 +1,98 @@
+from pprint import pprint
+from os import getenv
+import json
+import requests
+
+from .base import Base
+from ..utils.spec import build_spec_for_block
+
+
+class BlockPublish(Base):
+
+    def run(self):
+        spec = self._fetch_spec()
+        release = self._fetch_release()
+        if self.options.get('--dry-run'):
+            pprint("Specification that will be published:")
```

Comment:
this looks weird, apparently `pprint` wraps strings in quotes because.... quotes are pretty? Anyway, this should be just `print`